src/components/VideoListingCard.js

```diff
@@ -34,6 +36,17 @@ const VideoListingCard = ({ video }) => {
         }
     }
 
+    const updateHistory = async () => {
+        const updatedHistory = await removeFromHistory(video._id, user);
+        setHistory(updatedHistory);
+        notificationHandler("Removed from history");
+        try {
+
```

Comment:
try block is empty. it is intended or you planning to do write in future.

Reply:
fixed the empty try block.